region/maridia/inner-pink.json

```diff
@@ -1047,7 +1069,11 @@
                     "canSuitlessMaridia",
                     {"or": [
                       "canMochtroidClimb",
-                      "Grapple"
+                      "Grapple",
+                      {"and": [
+                        "canXrayClimb",
```

Comment:
For the same reason as Pants room, I think this should require `canSwim` or `canSuitlessMaridia`

Reply:
this one has it

Reply:
Oh, I missed that. Seems like this climb right now is only in with suitless though, not with Gravity.